demos/moderator/requirements-dev.txt

```diff
@@ -1,9 +1,9 @@
 -e .
-aiohttp==3.1.3
+aiohttp==3.7.3
```

Comment:
Looks like we're almost there now.
You'll need to change these back to 3.6.x as things are broken on 3.7 (which are causing the test failures).